### DISCUSSION TITLE
VC interested in talking to the founders. How to?

### DISCUSSION THREAD

**Discussion by OmarHedeya95** [FlowiseAI/Flowise#160] [Q&A]
Hey everyone,
Sounds completely random but could not find a better way to reach out to the founders of Flowise AI :D I am a VC and would love to talk with the founders. What would be the best way to reach you? :D

**Comment by HenryHengZJ** [comment]
hey feel free to drop in on discord!

**Reply by OmarHedeya95** [reply]
Got an email easier?
